src/sentry/replays/endpoints/organization_replay_count.py

```diff
@@ -23,6 +25,13 @@
 FILTER_HAS_A_REPLAY = "AND !replayId:''"
 
 
+class ReplayDataSourceValidator(serializers.Serializer):
+    data_source = serializers.ChoiceField(
+        choices=((Dataset.Discover.value), (Dataset.IssuePlatform.value)),
```

Comment:
do the values need to be wrapped in parentheses? i.e. would `choices=(Dataset.Discover.value, Dataset.IssuePlatform.value)` work?

Reply:
Looks like it works without the parentheses